README.md

```diff
@@ -50,8 +61,13 @@ this method should set the pizza's temperature to be 300. Note: you may not use
 the variable pizza inside your method.
 
 Your Answer:
-```js
-// write code here
+```
+var pizza = {
+  var temperature = 70;
+  var bake = function() {
+    temperature = 300;
```

Comment:
Here you are setting a global variable called temperature. To change the property `temperature` on the pizza object, we can use `pizza.temperature`, or even better, `this.temperature`. That last one works because `this` will refer to what the bake function is called on... so `pizza.bake()` will make `this` be `pizza` inside the function.